airflow/config_templates/config.yml

```diff
@@ -375,6 +375,21 @@ core:
       type: string
       example: "path.to.CustomXCom"
       default: "airflow.models.xcom.BaseXCom"
+    xcom_objectstorage_path:
+      description: |
+        Path to a location on object storage where XComs can be stored in url format.
+      version_added: 2.9.0
+      type: string
+      example: "s3://conn_id@bucket/path"
+      default: ""
+    xcom_objectstorage_threshold:
+      description: |
+          Threshold in bytes for storing XComs in object storage.
```

Comment:
Would be nice to specify behaviour for `-1` - from default it looks like it is intended for `-1` to mean no threasholdm but also 0 could be one (0 feels a bit more intuitive, but if it's explicitly stated that `-1` makes all data stored in object storage, then -1 is a also good. It's of course logically correct (with exceeds and figuring out what happens when you have 0-sized object), but spelling it out explicitly would save the mental effort to figure it out by the user.

Reply:
I think from reading your response threshold might be a misnomer. -1 signifies that everything goes to the database. 0 signifies that everything goes to object storage as any size of a variable will be large than 0. So opposite what you wrote.

Reply:
Then it's even more sign that it deserves spelling out expliciltly what it does.

Reply:
Yep. Done :-)